<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_difficulty</class>
 <widget class="QDialog" name="dialog_difficulty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_static">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择游戏难度！</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="but_dif0">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>简单</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="but_dif1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>普通</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="but_dif2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>困难</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_start">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_back">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
